### PR TITLE
Implement light theme

### LINKED ISSUE #393: Document theme in the docs

### LINKED ISSUE #394: Read dark mode preference from browser

### LINKED ISSUE #395: Create a theme context
vuex store

### LINKED ISSUE #396: Clean up components

### LINKED ISSUE #397: Persist theme choice on refresh

### LINKED ISSUE #398: Implement theme toggle button

### LINKED ISSUE #399: Implement theme styles from Figma file

### LINKED ISSUE #401: Add sitemap to README

### LINKED ISSUE #402: Create a sitemap

### LINKED ISSUE #432: store application theme name in the browser local storage
Currently, light-theme = true|false is stored in the local storage.  If the theme name is stored, this will allow more than just the light/dark theme for the future.

### PR DESCRIPTION
**Pull request by yuetloo**

### CODE REVIEW

**Comment by daodesigner** [comment]
Closes #405, closes #395 , closes #393  , closes #399 , closes #398 : 15 story points
@yuetloo please provide an estimate of time spent so far for sprint burndown. Keep up the great work!

**Comment by yuetloo** [comment]
@daodesigner, I spent 15 hours on this PR so far.

File: README.md [hunk 1]
```diff
@@ -144,6 +144,43 @@ yarn start:web
 yarn lint
 ```
 
+## Sitemap
+
+```
+/
```
Comment (daodesigner) [inline README.md]:
```
/
├─── /projects
│     └── /:id
├─── /round-information
├─── /rounds
│     └── /:address
├─── /about
│      ├──/maci
│      ├──/sybil-resistance
│      ├──/layer-2
│      ├──/how-it-works
│      │   ├──/contributors
│      │   └──/recipients
│      ├──/public-goods
│      ├──/quadratic-funding
│      └──/decentralization
├─── /recipients
├─── /verify
│      ├──/success/:hash?
│      └──/connect
├─── /join
│      ├──/success/:hash
│      ├──/project
│      ├──/fund
│      ├──/team
│      ├──/links
│      ├──/image
│      ├──/furthestStep
│      └──/summary
├─── /cart
└─── /transaction-success
       ├──/reallocation/:hash?
       └──/contribution/:hash?
```

File: vue-app/src/views/Verify.vue [hunk 2]
```diff
@@ -849,38 +853,10 @@ export default class VerifyView extends Vue {
   }
 }
 
-.loader {
```
Comment (daodesigner) [inline vue-app/src/views/Verify.vue:852]: where did these definitions move to?

**Review by daodesigner — CHANGES_REQUESTED** [review]

File: vue-app/src/views/Verify.vue (hunk 2 above)
Comment (yuetloo) [reply vue-app/src/views/Verify.vue:852]: they were defined in Loader.vue, exactly the same, so, I removed it here.

**Pull request merged**